content/docs/clients/javascript/index.md

```diff
@@ -7,42 +7,56 @@ The Exceptionless JavaScript client SDK makes it easy to report errors, log deta
 
 ---
 
-If you are looking for framework-specific guides, you can jump right to them below.
+Full guides can be found below:
 
-* React
-* Vue
-* Svelte
+* [Browser (no framework)](guides/javascript-example.md)
+* [React](./guides/react-example.md)
+* [Node](./guides/node-example.md)
+* [Express](./guides/express-example.md)
 
 ---
 
-To get started, you'll need to install the Exceptionless client. There are two main ways to do this: npm and CDN.
+This quickstart focuses on the vanilla JavaScript implementation of Exceptionless. 
 
-#### npm
+## npm
 
-To install with npm, run: `npm install @exceptionless/node --save`
+To install with npm, run: `npm install @exceptionless/browser`
+
+Next, you just need to call startup during your apps startup to automatically capture unhandled errors.
 
 ```js
-import { Exceptionless } from "@exceptionless/node";
+import { Exceptionless } from "@exceptionless/browser";
+// Or import { Exceptionless } from "https://unpkg.com/@exceptionless/browser";
 
-await Exceptionless.startup(c => {
-  c.apiKey: 'API_KEY_HERE'
+await Exceptionless.startup((c) => {
+  c.apiKey = "API_KEY_HERE";  
 });
```

Comment:
If we are trying to do simple here, then we should probably use the simplest form of startup `await Exceptionless.startup("API_KEY_HERE");`

Reply:
+1 to this. Updating this afternoon.